dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/S3Utils.java

```diff
@@ -293,15 +339,28 @@ private void downloadDirectory(String  tenantCode, String keyPrefix, String srcP
         }
     }
 
-    public void checkBucketNameIfNotPresent(String bucketName) {
-        if (!s3Client.doesBucketExistV2(bucketName)) {
-            logger.info("the current regionName is {}", s3Client.getRegionName());
-            s3Client.createBucket(bucketName);
+    public void checkBucketNameExists(String bucketName) {
+        if (StringUtils.isBlank(bucketName)) {
+            Stopper.stop();
```

Comment:
I think we should not stop in here.

Reply:
I think stop retry thread is not a good idea. If the user create the bucket during the retry. It should succeed instead of stop.